js/script.js

```diff
@@ -56,10 +55,13 @@ function refreshTable() {
             }
         }
         tableBody.appendChild(row);
-        filterTable();

```

Comment:
it seems that at this point we might have all filters active but we ignore them and display all the rows of the table, which might be confusing for a user .
Maybe we should reset filters ?